wm8960.c

```diff
@@ -1352,7 +1352,7 @@ static struct snd_soc_dai_driver wm8960_dai = {
 		.rates = WM8960_RATES,
 		.formats = WM8960_FORMATS,},
 	.ops = &wm8960_dai_ops,
-	.symmetric_rates = 1,
+	.symmetric_rate = 1,
```

Comment:
This should rather be (to preserve compatibility with previous kernel versions):
```
#if LINUX_VERSION_CODE < KERNEL_VERSION(5,15,0)
        .symmetric_rates = 1,
#else
        .symmetric_rate = 1,
#endif
```